src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -66,6 +67,23 @@ def transform_vm_list(vm_list):
     return [transform_vm(v) for v in vm_list]
 
 
+def transform_av_set_output(av_set):
+    # workaround till compute api version gets to 2017-04-30
+    if hasattr(av_set, 'sku') and hasattr(av_set.sku, 'name'):
+        setattr(av_set.sku, 'managed', av_set.sku.name == 'Aligned')
+        del av_set.sku.name
+    return av_set
+
+
+def transform_av_set_collection_output(av_sets):
```

Comment:
yep, because python 's argument is `pass by ref`, so let me just reuse the `transform_av_set_output`